GVRf/Framework/framework/src/main/jni/objects/components/render_data.cpp

```diff
@@ -36,9 +36,14 @@ const RenderPass* RenderData::pass(int pass) const {
 }
 
 void RenderData::set_mesh(Mesh* mesh) {
-    mesh_ = mesh;
-    mesh->add_dirty_flag(dirty_flag_);
-    *dirty_flag_ = true;
+    if (mesh != nullptr) {
```

Comment:
This nullptr check prevents in the crash in NativePhysics3DWorld.step. Though I believe the better fix is to synchronize all access to GVRWorld.mRigidBodies.